src/vm/byond.service.ts

```diff
@@ -131,6 +131,10 @@ export class ByondService {
     void this.lock.run(async () => {
       for await (const version of (await this.getByondFolder()).keys()) {
         this._versions.set(version, VersionStatus.Fetched);
+        // If this is the first version in the list, load it
+        if (this._activeVersion == null) {
+          await this.load(version, true);
```

Comment:
I'm not sure how meaningful the status of first installed version is. In fact, it's probably the oldest version and the one the user is the least likely to want

Reply:
i just want /some/ version to be actively loaded
so you don't have to click 'set active' every single time

the user can manage their own versions